pavelib/quality.py

```diff
@@ -272,7 +272,12 @@ def run_jshint(options):
     jshint_report = jshint_report_dir / "jshint.report"
     _prepare_report_dir(jshint_report_dir)
 
-    jshint_directories = ["common/static/js", "cms/static/js", "lms/static/js"]
+    jshint_directories = [
```

Comment:
Ah yes I'd love to know if this works too. I had initially put that code in with the intention of improving it later. :\